.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: build
+
+on:
+  push:
+
+env:
+  CURL_CACHE_DIR: ~/.cache/curl
+  GO_CACHE_DIR: ~/.cache/go-cache
+  PIP_CACHE_DIR: ~/.cache/pip
+  PRE_COMMIT_CACHE_DIR: ~/.cache/pre-commit
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cisagov/setup-env-github-action@develop
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Cache pip test requirements
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.PIP_CACHE_DIR }}
+          key: "${{ runner.os }}-pip-test-\
+            ${{ hashFiles('**/requirements-test.txt') }}"
+          restore-keys: |
+            ${{ runner.os }}-pip-test-
+            ${{ runner.os }}-pip-
+      - name: Cache pre-commit hooks
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.PRE_COMMIT_CACHE_DIR }}
+          key: "${{ runner.os }}-pre-commit-\
+            ${{ hashFiles('**/.pre-commit-config.yaml') }}"
+      - name: Cache curl downloads
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.CURL_CACHE_DIR }}
+          key: "${{ runner.os }}-curl-\
+            terraform-${{ env.TERRAFORM_VERSION }}"
+      - name: Cache go
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.GO_CACHE_DIR }}
+          key: "${{ runner.os }}-go"
+      - name: Install Terraform
+        run: |
+          mkdir -p ${{ env.CURL_CACHE_DIR }}
+          TERRAFORM_ZIP="terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
+          curl -v --output ${{ env.CURL_CACHE_DIR }}/${TERRAFORM_ZIP} \
```

Comment:
I think it's worth seeing if something like `\"...\"` will work here, since a space appearing in these expansions is a definite danger.

Reply:
`\"` didn't work.  I was able to quote the non-expanding portion of the expression.